### PR TITLE
Updates for ControlPlane deploy

### LINKED ISSUE #532: Migrate from Heroku to Control Plane
Update the project to be ready to be deployed on Control Plane.

### PR DESCRIPTION
**Pull request by ahangarha**
Known issue: Control Plane doesn't persist the database. Though we want this, we need to recreate the database again. This is WIP.

Closes #532 

This change is [Reviewable](https://reviewable.io/reviews/shakacode/react-webpack-rails-tutorial/548)

### CODE REVIEW

**Comment by justin808** [comment]
@ahangarha can we merge?

File: .controlplane/Dockerfile [hunk 1]
```diff
@@ -17,18 +17,20 @@ RUN bundle config set without 'development test' && \
...
+# COPY Gemfile* config.ru Rakefile babel.config.js postcss.config.js ./
+# COPY app ./app
+# COPY bin ./bin
+# COPY client ./client
+# COPY config ./config
+# COPY db ./db
+# COPY lib ./lib
+# COPY public ./public
+
+COPY . ./
```
Comment (ahangarha) [inline .controlplane/Dockerfile:33]: Either we should copy all files (similar to the old Docker file at the root of the project) or we should mention all the config files in the root directory.

I intentionally kept the old lines commented to decide about it.

**Review by justin808 — APPROVED** [review]

File: .controlplane/Dockerfile (hunk 1 above)
Comment (justin808) [reply .controlplane/Dockerfile:33]: do this in a followup

Comment (justin808) [reply .controlplane/Dockerfile:33]: if cannot be done within 12 hours

Comment (ahangarha) [reply .controlplane/Dockerfile]: Done

**Comment by justin808** [comment]
@Judahmeek @ahangarha  can you fix failing test in separate PR?

**Comment by ahangarha** [comment]
@justin808 The failing CI is a typical issue with our tests. We need to fix our tests. The number of false failings is not normal in this repo. We already have an issue for this: #543.
I just rerun the test and it passed.

This is not an issue for this project but we should fix it once.

**Pull request merged**